packages/pool/contracts/DelegationUtils.sol

```diff
@@ -42,22 +42,22 @@ contract DelegationUtils is RewardUtils, IDelegationUtils {
         if (userDelegate != address(0)) {
```

Comment:
I didn't want to emit `Delegated` because there is no delegation being made there. Maybe it should
```
require(userDelegate != delegate);
```
instead, what do you think?

Reply:
Created an issue, let's deal with it outside of this PR
https://github.com/api3dao/api3-dao/issues/148

Reply:
> Created an issue, let's deal with it outside of this PR
> #148

+1